============================= test session starts ==============================
platform linux -- Python 3.10.6, pytest-7.3.1, pluggy-1.0.0
rootdir: /data/sandeep/leetcode-master/.verifaiws
collected 1 item

204_Count_Primes/test_204_Count_Primes.py F                              [100%]

=================================== FAILURES ===================================
_______________________________ test_countPrimes _______________________________

    def test_countPrimes():
        sol = Solution()
    
        assert sol.countPrimes(10) == 4
        assert sol.countPrimes(5) == 2
        assert sol.countPrimes(25) == 9
        assert sol.countPrimes(0) == 0
    
        with pytest.raises(TypeError):
            sol.countPrimes("a")
            sol.countPrimes(None)
    
>       with pytest.raises(ValueError):
E       Failed: DID NOT RAISE <class 'ValueError'>

204_Count_Primes/test_204_Count_Primes.py:16: Failed
=========================== short test summary info ============================
FAILED 204_Count_Primes/test_204_Count_Primes.py::test_countPrimes - Failed: ...
============================== 1 failed in 0.01s ===============================
